__tests__/app.test.js

```diff
@@ -142,3 +142,27 @@ describe('PATCH/api/articles/article_id', () => {
     })
 
 })
+describe('GET/api/users', () => {
+    test('status 200 and returns an array of objects with appropriate properties', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.users).toBeInstanceOf(Array)
+            expect(body.users).toHaveLength(4)
+            body.users.forEach((user) => {
+                expect(user.username).toEqual(expect.any(String));
+                expect(user.name).toEqual(expect.any(String));
+                expect(user.avatar_url).toEqual(expect.any(String)); 
+            })
+        })
+    })
+    test('404 status code with error message when using invalid path', () => {
```

Comment:
This is the same test as line 34. It's a valid test but isn't particular to an ednpoint. I'd pop it in a stand alone describe block and remove the duplicate.